packages/iris-grid/src/IrisGridPartitionSelector.tsx

```diff
@@ -173,12 +178,16 @@ class IrisGridPartitionSelector<T> extends Component<
     return (
       <div className="iris-grid-partition-selector">
         <div className="status-message">
-          <span>Filtering &quot;{columnName}&quot; partition to</span>
+          <span>Filtering &quot;{column.name}&quot; partition to</span>
         </div>
         <div className="input-group">
           <input
             type="text"
-            value={partition}
+            value={
+              column.type === 'char' && partition.toString().length > 0
+                ? String.fromCharCode(parseInt(partition, 10))
+                : partition
+            }
```

Comment:
Might be able to use `IrisGridUtils.convertValueToText` here instead.

Reply:
I looked into it and `IrisGridUtils.convertValueToText` doesn't convert `partition` to a `char` because partition is always a `string` type and `convertValueToText` only performs the conversion for `number` types.